### ISSUE TITLE
Fine tune on my own dataset

### ISSUE THREAD

**Issue by USTClj** [yhenon/keras-frcnn#80]
HI! Thank you for your great work.  I flowed your guide  #64  to load pre-trained  weight. 
I put the model_frcnn.hdf5 (https://drive.google.com/file/d/0B2XPZBSClSzhdENkRmM1alVsN3c/view) to a dictionary named weight, and run 

python train_frcnn.py -o simple -p data/my_data.txt --input_weight_path weight

but the performance still seems poor (unsatisfied allocation rate=0.0824717) for some epochs.

When I run test.py, I got this:
00012.ppm
Elapsed time = 1.21947693825
[]
00022.ppm
Elapsed time = 0.416192054749
[]
00038.ppm
Elapsed time = 0.423347949982
[]

I simply change the class to 2 (one objection and background)
My datasets consists of 900 samples.

Could you give me some suggestion? Thank you

**Comment by yhenon** [comment]
Can you share the output from the training? It's hard to tell without seeing what the loss is like.

**Comment by USTClj** [comment]
Thank you for your replying. I have figured out this issue. The main cause is that the training isn't enough.
Perhaps, due to the shortage of small dataset, the detector network is likely to detect a lot of background.

**Comment by yhenon** [comment]
No problem, I'm working on adding more data augmentation which might help.
